msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -402,47 +410,50 @@ items:
 
     1. Select **Provision**.
 
+Doc-Improvement-Updating-alt-test-for-images9
```

Comment:
Removing branch name from within content `Doc-Improvement-Updating-alt-test-for-images9`.